proto/src/determined/experiment/v1/searcher.proto

```diff
@@ -4,26 +4,29 @@ package determined.experiment.v1;
 option go_package = "github.com/determined-ai/determined/proto/pkg/experimentv1";
 
 import "google/protobuf/struct.proto";
+import "protoc-gen-swagger/options/annotations.proto";
 
-// WorkloadKind defines the kind of workload that should be executed by trial
-// runners.
-enum Unit {
-  // Denotes an unknown Uz.
-  UNIT_UNSPECIFIED = 0;
-  // Unit type "Batches".
-  UNIT_BATCHES = 1;
-  // Unit type "Records".
-  UNIT_RECORDS = 2;
-  // Unit type "Epochs".
-  UNIT_EPOCHS = 3;
-}
-
-// TrainingUnits represents a single runnable operation emitted by a searcher.
-message TrainingUnits {
```

Comment:
technically breaking but .. i'm fine with breaking the rest api for preview search if you are, to make it better

Reply:
I personally don't care but I do think you should float this past #infra-ag.

Reply:
Done.

Reply:
I think we're fine here.